### ISSUE TITLE
Image links broken in README

### ISSUE THREAD

**Issue by re1** [NerdyPepper/scientifica#33]

**Comment by NerdyPepper** [comment]
ah it was about time those expired. ill add some new screenshots along with the doubled version.

if you use scientifica, feel free to submit some screenshots :^)

**Comment by CraftedCart** [comment]
Spacemacs/Emacs
![image](https://user-images.githubusercontent.com/5341220/55515951-deba1980-5663-11e9-8bee-d44fe9d8d19f.png)

lxappearance/GTK
![image](https://user-images.githubusercontent.com/5341220/55516071-250f7880-5664-11e9-8091-3a84733785bb.png)

My terminal - tmux + zsh + oh-my-zsh + bullet train theme + zsh-syntax-highlighting
![image](https://user-images.githubusercontent.com/5341220/55516421-11b0dd00-5665-11e9-9d55-f3850088be8f.png)

**Comment by cata0309** [comment]
Is there any chance you would want to help me make the terminal I use to look like yours ? I am really interested.

**Comment by cata0309** [comment]
@CraftedCart  ??

**Comment by NerdyPepper** [comment]
it looks like he has shifted to gitlab. you can find his [dotfiles](https://gitlab.com/CraftedCart/dotfiles) there.

**Comment by cata0309** [comment]
Thank you

**Comment by cyberglot** [comment]
<img width="1792" alt="Screenshot 2020-03-11 at 12 12 55" src="https://user-images.githubusercontent.com/609862/76427120-d3a75780-63a3-11ea-82f0-8540b0aa94a6.png">

**Comment by JesMJM** [comment]
Looks nice, it lets me fit more in less space while still being readable

![image](https://user-images.githubusercontent.com/30764221/79202099-162adc80-7e07-11ea-8fdd-e1827193c079.png)

**Comment by cata0309** [comment]
@CraftedCart is that the default theme on Spacemacs/Emacs?

**Comment by NiLuJe** [comment]
For something a wee bit more out there, I bundled scientifica in a [library](https://github.com/NiLuJe/FBInk) I wrote to handle printing stuff to consumer eInk devices, and we're using it to display OTA update progress in [KOReader](https://github.com/koreader/koreader):

[![KOReader + FBInk + scientifica](https://user-images.githubusercontent.com/202757/74514732-7d163d00-4f0d-11ea-9d53-15d01ce4238a.jpg)](https://github.com/koreader/koreader/releases/tag/v2020.02)

**Comment by CraftedCart** [comment]
> @CraftedCart is that the default theme on Spacemacs/Emacs?

@cata0309
Yeah, the default Spacemacs dark theme, with rainbow-mode to color each symbol differently.

**Comment by cata0309** [comment]
Does anybody know how to enable bitmap fonts on Linux systems without breaking any settings regarding the non bitmap fonts like antialiasing, scalling, hinting?
I tried some tutorials on how to do that but they aren't usable in editors like VSCode

**Comment by NerdyPepper** [comment]
@NiLuJe that is so cool! I'm glad you find it useful.

**Comment by hplar** [comment]
> Does anybody know how to enable bitmap fonts on Linux systems without breaking any settings regarding the non bitmap fonts like antialiasing, scalling, hinting?
> I tried some tutorials on how to do that but they aren't usable in editors like VSCode

I had an issue where I couldn't use Scientifica in PyCharm, turns out I needed to install the .ttf version to get PyCharm to recognize the font. Maybe you are running into the same issue with VSCode?

**Comment by execb5** [comment]
Damn, this font is just what I needed.

![2020-04-16-203845_1920x1080_scrot](https://user-images.githubusercontent.com/6979113/79516639-53b78180-8022-11ea-89f3-463c4329c860.png)

I still need to see how to setup everything correctly in urxvt

![out](https://user-images.githubusercontent.com/6979113/79516375-97f65200-8021-11ea-9af8-9f7dd3b7f459.gif)

**Comment by fxkrait** [comment]
@cyberglot Is that Emacs theme you're using perhaps this: [emacs-purple-haze-theme](https://github.com/emacsfodder/emacs-purple-haze-theme)? That's the closest I could find, and I didn't see any dotfiles where you reference said theme.

**Comment by cyberglot** [comment]
@greghab It is purple haze, but these days I use [rebecca](https://github.com/vic/rebecca-theme).

**Comment by aerphanas** [comment]
![Image](https://github.com/user-attachments/assets/0a6e0f4f-8b25-4455-9f3e-bf3e57de2b63)

**Comment by mgytr** [comment]
<img width="3840" height="2160" alt="Image" src="https://github.com/user-attachments/assets/38e1c236-e94c-411c-a117-226bd8f05e6f" />
